default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	"workbench.quickOpen.closeOnFocusLost": true,
+
+	// Controls whether the last typed input to Quick Open should be restored when opening it the next time.
+	"workbench.quickOpen.preserveInput": false,
+
+	// Controls whether the workbench should render with fewer animations.
+	//  - on: Always render with reduced motion.
+	//  - off: Do not render with reduced motion
+	//  - auto: Render with reduced motion based on OS configuration.
+	"workbench.reduceMotion": "auto",
+
+	// Controls the hover feedback delay in milliseconds of the dragging area in between views/editors.
+	"workbench.sash.hoverDelay": 300,
+
+	// Controls the feedback area size in pixels of the dragging area in between views/editors. Set it to a larger value if you feel it's hard to resize views using the mouse.
+	"workbench.sash.size": 4,
+
+	// Determines which settings editor to use by default.
+	//  - ui: Use the settings UI editor.
+	//  - json: Use the JSON file editor.
+	"workbench.settings.editor": "ui",
+
+	// Controls whether to enable the natural language search mode for settings. The natural language search is provided by a Microsoft online service.
+	"workbench.settings.enableNaturalLanguageSearch": true,
+
+	// Controls whether opening keybinding settings also opens an editor showing all default keybindings.
+	"workbench.settings.openDefaultKeybindings": false,
+
+	// Controls whether opening settings also opens an editor showing all default settings.
+	"workbench.settings.openDefaultSettings": false,
+
+	// Controls the behavior of the settings editor Table of Contents while searching.
+	//  - hide: Hide the Table of Contents while searching.
+	//  - filter: Filter the Table of Contents to just categories that have matching settings. Clicking a category will filter the results to that category.
+	"workbench.settings.settingsSearchTocBehavior": "filter",
+
+	// Controls whether to use the split JSON editor when editing settings as JSON.
+	"workbench.settings.useSplitJSON": false,
+
+	// Controls the location of the primary side bar and activity bar. They can either show on the left or right of the workbench. The secondary side bar will show on the opposite side of the workbench.
+	"workbench.sideBar.location": "left",
+
+	// Controls which editor is shown at startup, if none are restored from the previous session.
+	//  - none: Start without an editor.
+	//  - welcomePage: Open the Welcome page, with content to aid in getting started with VS Code and extensions.
+	//  - readme: Open the README when opening a folder that contains one, fallback to 'welcomePage' otherwise. Note: This is only observed as a global configuration, it will be ignored if set in a workspace or folder configuration.
+	//  - newUntitledFile: Open a new untitled file (only applies when opening an empty window).
+	//  - welcomePageInEmptyWorkbench: Open the Welcome page when opening an empty workbench.
+	"workbench.startupEditor": "welcomePage",
```

Comment:
I think this should be `"none"`.

Reply:
🤔 it looks like `"readme"` wouldn't work since it's ignored as a workspace setting

> Note: This is only observed as a global configuration, it will be ignored if set in a workspace or folder configuration.

I guess it's only for the README option?